tests/components/rfxtrx/test_cover.py

```diff
@@ -90,19 +95,22 @@ async def test_several_covers(hass, rfxtrx):
     )
     await hass.async_block_till_done()
 
-    assert 3 == len(rfxtrx_core.RFX_DEVICES)
-    device_num = 0
-    for id in rfxtrx_core.RFX_DEVICES:
-        entity = rfxtrx_core.RFX_DEVICES[id]
-        assert entity.signal_repetitions == 3
-        if entity.name == "Living":
-            device_num = device_num + 1
-        elif entity.name == "Bath":
-            device_num = device_num + 1
-        elif entity.name == "Test":
-            device_num = device_num + 1
+    state = hass.states.get("cover.test")
+    assert state
+    assert state.state == "closed"
+    assert state.attributes.get("friendly_name") == "Test"
 
-    assert 3 == device_num
+    state = hass.states.get("cover.bath")
+    assert state
+    assert state.state == "closed"
+    assert state.attributes.get("friendly_name") == "Bath"
+
+    state = hass.states.get("cover.living")
+    assert state
+    assert state.state == "closed"
+    assert state.attributes.get("friendly_name") == "Living"
+
+    assert len(hass.states.async_all()) == 3
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5802827)